### PR TITLE
bpo-40836: Add docstring to logging.fatal() and logging.Logger.fatal()

### PR DESCRIPTION
**Pull request by remilapeyre**
https://bugs.python.org/issue40836

Automerge-Triggered-By: @vsajip

### CODE REVIEW

**Review by auvipy — APPROVED** [review]
looks good

**Review by vsajip — COMMENTED** [review]
I think the docstrings should just say "don't use this method/function - use critical() instead." There is no point documenting `fatal()` further, as we don't want it to be used.

**Comment by remilapeyre** [comment]
> I think the docstrings should just say "don't use this method/function - use critical() instead." There is no point documenting `fatal()` further, as we don't want it to be used.

At least it's clear :) I changed it.

**Review by vsajip — CHANGES_REQUESTED** [review]
Please change as per earlier comment (I hit the Comment button by mistake before completing).

File: Lib/logging/__init__.py [hunk 1]
```diff
@@ -2052,11 +2045,7 @@ def critical(msg, *args, **kwargs):
 
 def fatal(msg, *args, **kwargs):
     """
-    Log a message with severity 'CRITICAL' on the root logger. If the logger
-    has no handlers, call basicConfig() to add a console handler with a
-    pre-defined format.
-
-    The critical() function is preferred.
+    Don't use this method, use critical() instead.
```
Comment (vsajip) [inline Lib/logging/__init__.py]: Should say "function" here rather than "method".

**Review by vsajip — CHANGES_REQUESTED** [review]

File: Lib/logging/__init__.py (hunk 1 above)
Comment (remilapeyre) [reply Lib/logging/__init__.py]: Yes, I fixed it.

**Comment by remilapeyre** [comment]
I have made the requested changes; please review again

**Review by vsajip — APPROVED** [review]

**Pull request merged**